src/supervisor/message_handlers.py

```diff
@@ -62,9 +61,10 @@ class EmailHandler(MessageHandler):  # pylint: disable=too-few-public-methods
         gzip input_path into a MIMEApplication object
     """
 
-    def __init__(self, email_settings, project_name=''):
+    def __init__(self, email_settings, client_name='', client_version=''):
```

Comment:
**question**: should these be given default values like `forgetful subordinate` and `0.0.0`